packages/create-cloudflare/src/frameworks/next/index.ts

```diff
@@ -167,3 +181,11 @@ const config: FrameworkConfig = {
 	compatibilityFlags: ["nodejs_compat"],
 };
 export default config;
+
+const getAppDirPath = (projectName: string) => {
+	try {
```

Comment:
No need to try-catch here any more. Just return the result from `probePaths()`.

Reply:
ah yeah sorry I missed this 😓